src/main/java/frc/robot/Robot.java

```diff
@@ -287,27 +289,43 @@ public void robotInit() {
         .x()
         .whileTrue(
             Commands.parallel(
-                shooter.runFlywheelVoltageCmd(Rotation2d.fromDegrees(60.0), -5.0),
+                shooter.runFlywheelVoltageCmd(Rotation2d.fromDegrees(30.0), -5.0),
```

Comment:
to help get out of jams we tilt the shooter up when rejecting pieces, this change is just to make it less high to avoid stalling when the shooter isnt zeroed